molecule/postgrespro/vars/postgrespro_vars.yml

```diff
@@ -10,7 +10,7 @@ postgresql_bin_dir: "/opt/pgpro/std-{{ postgresql_version }}/bin"
 postgresql_unix_socket_dir: "/tmp"
 
 # YUM Repository
-_baseurl:
+_baseurl: # yamllint disable rule:line-length
```

Comment:
@vitabaks 

In the end, I have decided to ignore certain lengths (especially URLs), as it will indeed be simpler that way

However, we will still keep the linter so that it can flag any issues in each new pull request, allowing us to handle them on a case-by-case basis